### PR TITLE
patch cves

### PR DESCRIPTION
**Pull request by davejrt**
remediates the following CVE's:

https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-36222 (krb5-libs)
https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-22922 (curl)
https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-22924 (curl)

### CODE REVIEW

File: docker-images/minio/build.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 set -ex
 cd "$(dirname "${BASH_SOURCE[0]}")"
 
-# Retag the 2021-04-22 MinIO release
-MINIO_RELEASE="RELEASE.2021-04-22T15-44-28Z"
+# Retag the 2021-08-31 MinIO release
+MINIO_RELEASE="RELEASE.2021-08-31T05-46-54Z"
```
Comment (efritz) [inline docker-images/minio/build.sh:7]: cmd/server also has this version.

**Review by efritz — APPROVED** [review]

File: docker-images/minio/build.sh (hunk 1 above)
Comment (efritz) [reply docker-images/minio/build.sh:7]: https://k8s.sgdev.org/search?q=context:global+ENV+MINIO_VERSION%3DRELEASE.2021-04-22T15-44-28Z&patternType=literal

**Comment by davejrt** [comment]
@efritz can you confirm if there are any implications in bumping the minio image here to remediate the following:
https://cve.mitre.org/cgi-bin/cvename.cgi?name=RHSA-2021:2170
https://access.redhat.com/errata/RHSA-2021:2717

File: docker-images/minio/build.sh (hunk 1 above)
Comment (davejrt) [reply docker-images/minio/build.sh:7]: TY!

**Comment by efritz** [comment]
@davejrt I don't see any issues upgrading MinIO, we use pretty basic features and unless they're releasing major regressions it should be fine.

The codeintel-qa pipeline uses the server image that bundles MinIO (be sure to update the version in that build as well - see my comment inline), so if there is a major issue we'll catch it in that workflow triggered off of `main`.

**Review by bobheadxi — APPROVED** [review]
thank you!

File: docker-images/minio/build.sh (hunk 1 above)
Comment (davejrt) [reply docker-images/minio/build.sh:7]: https://github.com/sourcegraph/sourcegraph/pull/24465/commits/a2a4d2f0ec136d4573b012fad21e16c2770eaf2c

**Review by efritz — APPROVED** [review]

**Pull request merged**